src/main/java/no/nav/familie/integrasjoner/dokarkiv/metadata/BarnetrygdVedtakMetadata.kt

```diff
@@ -0,0 +1,18 @@
+package no.nav.familie.integrasjoner.dokarkiv.metadata
+
+import no.nav.familie.integrasjoner.dokarkiv.client.domene.JournalpostType
+import org.springframework.stereotype.Component
+
+@Component
+object BarnetrygdVedtakMetadata : DokumentMetadata {
+    override val journalpostType: JournalpostType = JournalpostType.UTGAAENDE
+    override val fagsakSystem: String?= "FS36" //TODO: find out a real id
+    override val tema: String = "BAR"
+    override val behandlingstema: String? = "ab0270" // https://confluence.adeo.no/display/BOA/Behandlingstema
+    override val kanal: String? = ""
```

Comment:
Do we even need to override this?

Reply:
yes, as an implementation class it must override all variables from the interface